app/views/shopping_lists/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render 'shared/navigation' %>
+    <h1 class= "shopping-list-heading">Shopping List</h1>
+      <div class="display">  
+      <p class="total-amount">Amount of food item to buy: $<%= @total_price %></p> <p class="total-amount">Total value of food needed:<%= @total_items %></p>
+    </div>
+    <table>
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th>Quantity</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @shoppinglist.each do |food| %>
+        <% unless food[:price].zero? %>
+          <tr>
+            <th><%= food[:food].name %></th>
+            <th><%= food[:quantity] %><%= food[:food].measurement_unit %></th>
+            <th>$ <%= food[:price] %></th>
+          </tr>
+        <% end %>
+      <% end %>
+    </tbody>
+    </table>
```

Comment:
- [ ] @Newtayo and @danielmatama Great work adding a  shopping list page, however, am unable to generate a shopping list from my recipe list

`mylist`
![image](https://github.com/danielmatama/Recipe-app-rails/assets/61408860/17636db4-6e68-4f3e-a99b-bdef84ff02ab)

`shopping list page after I click generate shopping list button`
![image](https://github.com/danielmatama/Recipe-app-rails/assets/61408860/e9f7f9be-4a3f-4e36-a306-8edcd0d3bb3b)

Reply:
@danmainah Thank you for the feedback. However, the general shopping list will only generate a list if the quantity of food you need for your recipe is less than the quantity you have in your food list